lib/index.js

```diff
@@ -7,8 +7,7 @@ async function init(config, logger) {
 		await framework.init({config, logger});
```

Comment:
This already ensures that the plugin setup (incl. async tooling setup) is done before requests come in...